airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryUtils.java

```diff
@@ -203,12 +203,14 @@ public static Table createTable(final BigQuery bigquery, final String datasetNam
    */
   static void createPartitionedTableIfNotExists(final BigQuery bigquery, final TableId tableId, final Schema schema) {
     try {
+      final var chunkingColumn =
+          TypingAndDedupingFlag.isDestinationV2() ? JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT : JavaBaseConstants.COLUMN_NAME_EMITTED_AT;
       final TimePartitioning partitioning = TimePartitioning.newBuilder(TimePartitioning.Type.DAY)
-          .setField(JavaBaseConstants.COLUMN_NAME_EMITTED_AT)
+          .setField(chunkingColumn)
           .build();
 
       final Clustering clustering = Clustering.newBuilder()
```

Comment:
@evantahler this is where the clustering logic is, its an array so I wonder if clustering on all columns is crazy? or maybe just leaving off the data column?

Reply:
@alex-gron maybe you have some advice?  I don't _think_ we should be clustering on more than one column in BigQuery perhaps?  Either way, I don't think this is equivalent to what we are doing now - clustering by one column, COLUMN_NAME_EMITTED_AT

<img width="1955" alt="Screenshot 2023-07-05 at 3 10 22 PM" src="https://github.com/airbytehq/airbyte/assets/303226/bf9a418a-144b-4169-a844-9362d241ccd6">